docs/guides/hooks-list.rst

```diff
@@ -478,6 +478,12 @@ Embed
 
 **embed_get_upload_sections, all**
 
+Groups
+------
+
+**profile_buttons, group**
+	Filters buttons to be registered in the title menu of the group profile page
```

Comment:
They are arrays, not menu item instances.
On Dec 25, 2015 9:49 PM, "Steve Clay" <notifications@github.com> wrote:

> In docs/guides/hooks-list.rst
> <https://github.com/Elgg/Elgg/pull/9255#discussion_r48443279>:
>
> > @@ -478,6 +478,12 @@ Embed
> >
> >  **embed_get_upload_sections, all**
> >
> > +Groups
> > +------
> > +
> > +**profile_buttons, group**
> > +	Filters buttons to be registered in the title menu of the group profile page
>
> "Filters buttons (ElggMenuItem instances) to be registered ..."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9255/files#r48443279>.
>